mappers/modbus_mapper/deployment.yaml

```diff
@@ -30,10 +30,9 @@ spec:
         - name: dpl-config-volume
           mountPath: /opt/src/dpl
       nodeSelector:
-        modbus: "true"      
+        modbus: "true"
       volumes:
       - name: dpl-config-volume
         configMap:
           name:  device-profile-config-<edge_node_name>
       restartPolicy: Always
-      
```

Comment:
No newline at end of file?

Reply:
The vim would auto remove newline at end of file by setting, it is what text editor does, add it.